Update Chinese translation of "about" and "getting_started"

alaahong opened the pull request:
- website_and_docs/content/documentation/about
- website_and_docs/content/documentation/getting_started

**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.


<!--- Provide a general summary of your changes in the Title above -->

### Description
<!--- Describe your changes in detail -->

### Motivation and Context
<!--- Why is this change required? What problem does it solve? -->

### Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [ ] Code example added (and I also added the example to all translated languages)
- [X] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
- [X] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [X] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.
<!--- Provide a general summary of your changes in the Title above -->

alaahong committed:
Update Chinese translation of "about" and "getting_started"
- website_and_docs/content/documentation/about
- website_and_docs/content/documentation/getting_started

harsha509 commented on website_and_docs/content/documentation/getting_started/installing_browser_drivers.zh-cn.md:
```diff
@@ -1,24 +1,16 @@
 weight: 2
 description: >
-  Setting up your browser ready to be automated.
+设置您的浏览器用于自动化.
```
You need to provide one tab space in here. This is the reason build got failed AFAIK!

harsha509 commented on website_and_docs/content/documentation/getting_started/installing_selenium_libraries.zh-cn.md:
```diff
@@ -3,19 +3,10 @@ title: "安装 Selenium 库"
 weight: 1
 description: >
-  Setting up the Selenium library for your favourite programming language.
+为您喜爱的编程语言设置Selenium库.
```
Same as above!

alaahong replied:
Sorry for the mistake. let me configure latest hugo on my local to validate that

harsha509 reviewed (changes requested):
Hi @alaahong !

Thanks for the PR!

There is a build failure due to missing tab spaces in a couple of places. Could you please look into it

Thanks,
Harsha

alaahong committed:
Update Chinese translation of "about" and "getting_started"
- website_and_docs/content/documentation/about
- website_and_docs/content/documentation/getting_started
- website_and_docs/content/documentation/_index.zh-cn.md

harsha509 committed:
Merge branch 'dev' into feature/about_getting_started_20210907

harsha509 reviewed (approved):
Thank you @alaahong!
